packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -609,6 +611,7 @@ class _RailDestination extends StatelessWidget {
   final TextStyle labelTextStyle;
   final VoidCallback onTap;
   final String indexLabel;
+  final double padding;
```

Comment:
Why not just be an `EdgeInsetsGeometry`, similar to [Containers](https://master-api.flutter.dev/flutter/widgets/Container/padding.html)? This would make the padding property more flexible and consistent with how padding is defined elsewhere.

Reply:
Yes, I'll change it to accept ```EdgeInsetsGeometry``` :+1: